web/beacon-app/src/features/teams/api/invitationAuthenticationRequest.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiAdapters } from '@/application/api/ApiAdapters';
+import { getValidApiResponse, Request } from '@/application/api/ApiService';
+import { APP_ROUTE } from '@/constants';
+import { UserAuthResponse } from '@/features/auth';
+
+export function invitationAuthenticationRequest(
+  request: Request
+): ApiAdapters['getInvitationAuthentication'] {
+  return async (token: string) => {
+    const response = (await request(`${APP_ROUTE.INVITE}/accept`, {
+      method: 'POST',
+      data: {
+        token,
+      },
+    })) as any;
+
+    return getValidApiResponse<UserAuthResponse>(response);
```

Comment:
The `UserAuthResponse` is listed [here](https://github.com/rotationalio/ensign/blob/develop/web/beacon-app/src/features/auth/types/LoginService.ts).